Cheat Engine/launcher/cheatengine.lpr

```diff
@@ -34,6 +47,15 @@
 
     param: string;
     i: integer;
+
+procedure Launch(const exe: widestring);

```

Comment:
i put this here because i imagine pascal doesn't tolerate referencing not-yet-declared global variables

Reply:
on a second thought this doesn't need to be a procedure since it's at the end; we just assign exe to whatever